app/javascript/dashboard/App.vue

```diff
@@ -1,13 +1,13 @@
 <template>
   <div
-    v-if="!authUIFlags.isFetching"
+    v-if="!authUIFlags.isFetching && !accountUIFlags.isFetchingItem"
```

Comment:
Done to avoid flashing CTAs as the page would be loaded before the account information is available.